src/server/dstu2/observation/controllers/observation.controller.js

```diff
@@ -1,7 +1,9 @@
 const { ServerError } = require('../../../utils/error.utils');
 const { VERSIONS } = require('../../../../constants');
+const { Observation } = require('../../../resources/Patient');
```

Comment:
Is Observation in patient?

Also, since dstu2 exports resources it needs, you can do this:

```
// In observation controller
const { Observation } = require('../../resources');
// In patient controller
const { Patient } = require('../../resources');
```